components/user_settings/security/index.js

```diff
@@ -28,7 +28,7 @@ function mapStateToProps(state, ownProps) {
     const enableLdap = config.EnableLdap === 'true';
     const enableSaml = config.EnableSaml === 'true';
     const enableSignUpWithOffice365 = config.EnableSignUpWithOffice365 === 'true';
-    const experimentalEnableAuthenticationTransfer = config.ExperimentalEnableAuthenticationTransfer === 'true';
+    const experimentalEnableAuthenticationTransfer = config.ExperimentalEnableAuthenticationTransfer !== 'false';
```

Comment:
Ugh, I thought I had successfully scanned the 4.7 branch to find cases like this after @koxen found and fixed https://github.com/mattermost/mattermost-server/pull/8440. I ended up filing https://mattermost.atlassian.net/browse/MM-9804 so we can avoid this kind of implicit knowledge, but this is the simplest solution for 4.8.

Reply:
Hmm, my scan was accurate -- this bit was always checked as `ExperimentalEnableAuthenticationTransfer === 'true'` since the feature was originally introduced. The default, is supposed to be `false`, right? Is this a behaviour change then?

Reply:
We should definitely have a follow-up ticket for this to make sure that every setting is set regardless of whether a license is present

Reply:
(Discussed on pre-release: turns out to reproduce on 4.6, so the proposed behaviour change is correct. MM-9804 should solve this holistically.)